code/modules/projectiles/projectile.dm

```diff
@@ -549,7 +549,7 @@
 		var/mob/living/L = target
 		if(!direct_target)
 			var/checking = MOBILITY_USE | MOBILITY_STAND | MOBILITY_MOVE
-			if(!((L.mobility_flags & checking) == checking) || !(L.stat == CONSCIOUS))		//If they're able to 1. stand or 2. use items or 3. move, AND they are not softcrit,  they are not stunned enough to dodge projectiles passing over.
+			if(!(L.mobility_flags & checking) || L.stat == DEAD)		// If target not able to use items, move, or stand - or if they're just dead, pass over.

```

Comment:
I don't really get why you would have this stipulation at all, why would being dead make you ignore bullets or not?

Reply:
Also if they don't hit dead people why would they hit resting people? I am very confused at the sate of this PR

Reply:
I'm not changing anything, this is pretty much exactly how it was before.

Reply:
Why not just swap true and false on return?

Reply:
```suggestion
			if(!(L.mobility_flags & checking) || L.stat == DEAD)		// If target not able to use items, move and stand - or if they're just dead, pass over.
```
As i understand `!(L.mobility_flags & checking)` = 1  only when `L.mobility_flags` dont has any of `checking` flags in self

Reply:
!(L.mobility_flags & checking) = 1 if it doesn't have any of them, so it's equivalent to "or" no?

Reply:
from http://www.byond.com/docs/ref/index.html#/operator/&
If A and B are lists, the result is a list that contains only items that were in both lists, in the order of list A.
list(1,2) & list (2,3) = list(2) 
if( list(1,2) & list (2,3) ) return 1
if( ! (list(1,2) & list (2,3)) ) return 0

Reply:
Oh you're right, thanks